wizard/src/utils/detect_place.ts

```diff
@@ -74,13 +82,42 @@ export class PlaceDetector {
     }
   }
 
+  // The low confidence column detector simply checks if the column header
+  // (string) matches one of the supported place strings in this.placeTypes.
+  // The header is converted to lower case and only alphanumeric chars are used.
+  // If there is no match, the return value is null.
+  detectLowConfidence(header: string): string {
+    var h = header.toLowerCase().replace(/[^a-z0-9]/gi,'')
+
+    if (this.placeTypes.has(h)) {
```

Comment:
maybe instead of having to convert `h` to the place type string, `this.placeTypes` can be a map where key is the placeType in all lowercase and the value is the actual placeType string, then here we can just do `return this.placeTypes.has(h) ? this.placeTypes.get(h) : null`

Reply:
Excellent! Thanks. Done